charts/tidb-cluster/templates/monitor-deployment.yaml

```diff
@@ -61,6 +63,25 @@ spec:
         volumeMounts:
         - name: monitor-data
           mountPath: /data
+      - name: monitor-initializer
```

Comment:
I think it's better to merge the two init containers into one.

Reply:
Done